runtime/vm/JFRChunkWriter.cpp

```diff
@@ -676,5 +678,112 @@ VM_JFRChunkWriter::writeJVMInformationEvent()
 	return dataStart;
 }
 
+U_8 *
+VM_JFRChunkWriter::writePhysicalMemoryEvent()
+{
+	/* reserve size field */
+	U_8 *dataStart = _bufferWriter->getAndIncCursor(sizeof(U_32));
+
+	_bufferWriter->writeLEB128(PhysicalMemoryID);
+
+	/* write start time */
+	_bufferWriter->writeLEB128(j9time_current_time_millis());
+
+	J9MemoryInfo memInfo = {0};
+	I_32 rc = j9sysinfo_get_memory_info(&memInfo);
+	if (0 == rc) {
+		/* write total size */
+		_bufferWriter->writeLEB128(memInfo.totalPhysical);
+		/* write used size */
+		_bufferWriter->writeLEB128(memInfo.totalPhysical - memInfo.availPhysical);
```

Comment:
can you confirm that these are actually constant events? Do the value with the RI ever change? You may need to run a longer example

Reply:
Can confirm that the `usedSize` field is changing over time

Reply:
then this should be computed each time

Reply:
that's what we are doing right now - `j9sysinfo_get_memory_info` is called in `writePhysicalMemoryEvent`